lib/utils/download-results.js

```diff
@@ -0,0 +1,65 @@
+import exportImage from './export-svg';
+
+function generateCsvContent(data) {
+  let csvContent = 'data:text/csv;charset=utf-8,';
+
+  data.forEach((row, i) => {
+    row.forEach((cell, j) => {
+      csvContent += String(cell).replace(/,/g, '');
+      if (row.length > j + 1) {
+        csvContent += ',';
+      }
+    });
+    if (data.length > i + 1) {
+      csvContent += '\n';
+    }
+  });
+
+  return csvContent;
+}
+
+export default function downloadResults(obj) {
+  const { event, type, data } = obj;
+  event.preventDefault();
+  const supportedFormats = ['json', 'csv', 'jpg', 'jpeg', 'png'];
+  const format = type.toLowerCase();
+  if (!supportedFormats.includes(type)) {
+    throw new Error('This type is not supported');
+  }
+
+  const fileName = 'chart';
+  let fileFormat = '';
+  let content = '';
+
+  if (format === 'png') {
```

Comment:
json and csv download works, but png/jpg doesnt 

FileSaver.min.js:1 Uncaught (in promise) TypeError: Failed to execute 'createObjectURL' on 'URL': No function was found that matched the signature provided.
    at a (FileSaver.min.js:1)
    at export-svg.js:10